Diagnostics/Get-NetView.PS1

```diff
@@ -933,58 +933,64 @@ function MellanoxGetFiles {
         [parameter(Mandatory=$true)] [String] $OutDir
     )
 
-    $dir = Get-Item $OutDir -ErrorAction SilentlyContinue
-    if(!$dir) {
-        Write-Warning "'$NicName': Couldn't get the directory $OutDir"
-        return
-    }
+    $file = "MellanoxDetail.txt"
+    $out = Join-Path $OutDir $file
 
+    [String []] $cmds = @()
     $driverFileName = (Get-NetAdapter -name $NicName).DriverFileName
-    $driverName = $driverFileName.split('.')[0]
-    $path = [System.Environment]::ExpandEnvironmentVariables("%HOMEDRIVE%\Program Files\Mellanox\")
-    Switch("$driverName") {
-        "mlx5" {$path = $path + "MLNX_WinOF2"; break}
-        "mlnx5" {$path = $path + "MLNX_WinOF2_Azure"; break}
-        "mlnx5hpc" {$path = $path + "MLNX_WinOF2_Azure_HPC"; break}
+    $driverName     = $driverFileName.split('.')[0]
+    $path           = [System.Environment]::ExpandEnvironmentVariables("%HOMEDRIVE%\Program Files\Mellanox\")
```

Comment:
Environment variables can be directly accessed in PowerShell via `$env`. Also make sure to use the appropriate variable (`$env:ProgramFiles` for C:\Program Files, `$env:windir` for C:\Windows).

So this whole line can be replaced with:
```PowerShell
$path = "$env:ProgramFiles\Mellanox\"

Please also make this change elsewhere.